### PR TITLE
feat(site launch): add storybook for final state screens

### PR DESCRIPTION
**Pull request by kishore03109**
## Problem

The current site launch process does not have the final states screens. 

## Solution

This PR only focuses on introducing the storybook for these screens `Site Launch Success State` and `Site Launch Failure State`, the functionality will be tackled in upcoming PRs.

Screenshot 2023-08-10 at 11 59 17 AM

Safe to merge to dev since this feature is not accessed by anyone at the moment (behind a feature flag). 

### Success State 
Screenshot 2023-08-10 at 11 57 57 AM

### Failure State 
Screenshot 2023-08-10 at 11 58 17 AM

## Tests

nil -> this pr does not tackle functionality

### CODE REVIEW

**Comment by kishore03109** [comment]
!run-chromatic

File: src/layouts/SiteLaunchPad/components/SiteLaunchFinalState.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      <SiteLaunchFailureImage />
+
+      <Text
+        textStyle="body-1"
+        textColor="base.content.default"
+        mt="1.5rem"
+        textAlign="center"
+      >
+        The most common reason for site launch failure is unknown cloudflare
+        records. Please remove all existing domains and try site launch again.
```
Comment (alexanderleegs) [inline src/layouts/SiteLaunchPad/components/SiteLaunchFinalState.tsx:57]: nit: are we standardizing to `Cloudflare` haha

**Review by alexanderleegs — APPROVED** [review]

File: src/layouts/SiteLaunchPad/components/SiteLaunchFinalState.tsx (hunk 1 above)
Comment (kishore03109) [reply src/layouts/SiteLaunchPad/components/SiteLaunchFinalState.tsx:57]: omg is actually cloudfront this slipped my brain

Comment (kishore03109) [reply src/layouts/SiteLaunchPad/components/SiteLaunchFinalState.tsx]: thanks for the catch 7525244b16b989c3b82b68001e4d9914327cef44

**Review by harishv7 — APPROVED** [review]
lgtm, some conflicts pending

**Comment by kishore03109** [comment]
!run-chromatic

**Pull request merged**